data/crac-io/crac-io-json/src/main/java/com/powsybl/openrao/data/craciojson/JsonImport.java

```diff
@@ -50,18 +56,27 @@ public boolean exists(InputStream inputStream) {
     }
 
     @Override
-    public Crac importData(InputStream inputStream, CracFactory cracFactory, Network network) {
+    public CracCreationContext importData(InputStream inputStream, CracCreationParameters cracCreationParameters, Network network, OffsetDateTime offsetDateTime) {
         if (network == null) {
             throw new OpenRaoException("Network object is null but it is needed to map contingency's elements");
         }
+        if (offsetDateTime != null) {
+            // TODO : exception or warning
```

Comment:
Shall we discuss about this to decide whether to warn or throw an exception?

Reply:
I'll just add a warning (the json importer does not throw exceptions anymore, it would be strange to throw one just for this minor case)